ui-manchette/src/components/OperationalPoint.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { OperationalPointType } from '../types/pathPropertiesTypes';
+import '@osrd-project/ui-core/dist/theme.css';
+import { positionKm } from './helpers';
+
+const OperationalPoint: React.FC<OperationalPointType> = ({ extensions, id, part, position }) => {
+  return (
+    <div className="flex op items-baseline" id={id}>
+      <div className="op-position justify-self-start text-end">{positionKm(position)}</div>
+
+      <div className="op-name mr-2 ml-2 justify-self-start">{extensions?.identifier?.name}</div>
```

Comment:
Can you use mx-2 instead of mr-2 and ml-2 ? I tried, but it didn't work :thinking: 
```suggestion
      <div className="op-name mr-2 ml-2 justify-self-start">{extensions?.identifier?.name}</div>
```